mobify/progressive-web-scaffold
# Checkout Payment & Confirmation Pages

Finish mocking up the Payment and Confirmation pages' front end UI.

 **JIRA**: [WEB-1016](https://mobify.atlassian.net/browse/WEB-1016) [WEB-1017](https://mobify.atlassian.net/browse/WEB-1017)
**Mockups**: [Payment page](https://mobify.invisionapp.com/d/main#/console/9834411/210236445/preview#project_console), [Confirmation page](https://mobify.invisionapp.com/d/main#/console/9834411/210243064/preview#project_console)

## Changes
- Finish mocking up the Checkout Payment template
- Finish mocking up the Checkout Confirmation template

## Todo
- [x] Fix the shadow on `shrinkToContent` modals

## How to test-drive this PR
- Run `npm run dev`
- Add any items to the cart
- Skip most of the checkout and go straight to the [Payment page](https://www.merlinspotions.com/checkout/payment/)
- Visit the [Confirmation page](https://www.merlinspotions.com/checkout/confirmation/)
- Ensure that it matches the mockups as intended
